cmd/machine-config-daemon/pivot.go

```diff
@@ -79,6 +79,17 @@ func run(_ *cobra.Command, args []string) (retErr error) {
 		}
 	}
 
+	// Check to see if we need to tune kernel arguments
+	tuningChanged, err := daemon.UpdateTuningArgs(daemon.KernelTuningFile, daemon.CmdLineFile)
```

Comment:
since we're not using the kernel-args file anymore in 4.9+, I think this commit can be dropped too

Reply:
We use kernel-args for bootstrap

Reply:
I think now that https://github.com/openshift/installer/pull/5439 has merged, the installer doesn't use it anymore either. I think that was the last place?